models/POP/shell_scripts/cesm2_1/stitch_post.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Copyright 2020 University Corporation for Atmospheric Research
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.
+# Please view the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# ==============================================================================
+
+#Aim: stitch together filter output to make diagnostic files
```

Comment:
This needs a lot more commenting ... what directory do I run it in, the first real executable statement is 'remove all the netcdf files' ... that could be BAD ... how about some bulletproofing?